cobra.go

```diff
@@ -205,3 +206,17 @@ func stringInSlice(a string, list []string) bool {
 	}
 	return false
 }
+
+// CheckErr prints the msg with the prefix 'Error:' and exits with error code 1. If the msg is nil, it does nothing.
+func CheckErr(msg interface{}) {
+	if msg != nil {
+		fmt.Fprintln(os.Stderr, "Error:", msg)
+		os.Exit(1)
+	}
+}
+
+// WrStringAndCheck writes a string into a buffer, and checks if the error is not nil.
+func WrStringAndCheck(b io.StringWriter, s string) {
```

Comment:
Let's rename this to `WriteStringAndCheck`. Be descriptive w/o being overly wordy for exported functions and methods.